scripts/rollup/modules.js

```diff
@@ -136,14 +140,15 @@ function ignoreReactNativeModules() {
   ];
 }
 
-function getExternalModules(externals, bundleType, isRenderer) {
+function getExternalModules(externals, bundleType, moduleType) {
   // external modules tell Rollup that we should not attempt
   // to bundle these modules and instead treat them as
   // external depedencies to the bundle. so for CJS bundles
   // this means having a require("name-of-external-module") at
   // the top of the bundle. for UMD bundles this means having
   // both a require and a global check for them
   let externalModules = externals.slice();
+  const isRenderer = moduleType === RENDERER || moduleType === RECONCILER;
```

Comment:
This looks counter-intuitive. I would probably replace

```js
       if (isRenderer) {
         externalModules.push('react');
       }
```

with

```js
       if (moduleType !== ISOMORPHIC) {
         externalModules.push('react');
       }
```

below.